iot_backend_py/settings.py

```diff
@@ -37,13 +38,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'dashboard.apps.DashboardConfig'
 ]
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
Why?

Reply:
As we are dealing with RestAPIs only, there were two options - 
- Disabling CSRF for each and every endpoint Individually
- Or Disabling it globally

As all the endpoints are stateless so opted to remove it globally

Reply:
Okay, let's see this.